bee-ternary/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "bee-ternary"
-version = "0.3.4-alpha"
+version = "0.4-alpha"
```

```suggestion
version = "0.4.0-alpha"
```
Just for consistency with other crates

---

Whoops, this was accidental